src/Transactions/TransactionsList/TransactionsList.js

```diff
@@ -0,0 +1,182 @@
+const columnMapping = {
+  transactionDate: <FormattedMessage id="ui-finance.transaction.date" />,
+  transactionType: <FormattedMessage id="ui-finance.transaction.type" />,
+  amount: <FormattedMessage id="ui-finance.transaction.amount" />,
+  fromFundId: <FormattedMessage id="ui-finance.transaction.from" />,
+  toFundId: <FormattedMessage id="ui-finance.transaction.to" />,
+  source: <FormattedMessage id="ui-finance.transaction.source" />,
+  tagsList: <FormattedMessage id="ui-finance.transaction.tags" />,
+};
+const getResultsFormatter = (funds) => {
+  const fundsMap = funds.reduce((acc, fund) => {
+    acc[fund.id] = fund.code;
+
+    return acc;
+  }, {});
+
+  return ({
+    transactionDate: item => <FolioFormattedTime dateString={get(item, 'metadata.createdDate')} />,
+    transactionType: item => <FormattedMessage id={`ui-finance.transaction.type.${item.transactionType}`} />,
+    amount: item => (
+      <AmountWithCurrencyField
+        amount={item.amount}
+        currency={item.currency}
+      />
+    ),
+    fromFundId: item => fundsMap[item.fromFundId],
+    toFundId: item => fundsMap[item.toFundId],
+    source: item => <FormattedMessage id={`ui-finance.transaction.source.${item.source}`} />,
+    tagsList: item => get(item, 'tags.tagList', []).join(', '),
+  });
+};
+
+const TransactionsList = ({
+  onNeedMoreData,
+  resetData,
+  funds,
+  transactionsCount,
+  isLoadingTransactions,
+  transactions,
+  history,
+  location,
+  match,
+}) => {
+  const [
+    filters,
+    searchQuery,
+    applyFilters,
+    applySearch,
+    changeSearch,
+    resetFilters,
+  ] = useLocationFilters(location, history, resetData);
+  const [
+    sortingField,
+    sortingDirection,
+    changeSorting,
+  ] = useLocationSorting(location, history, resetData, sortableFields);
+
+  const selectedItem = useCallback(
+    (e, meta) => {
+      history.push({
+        pathname: `${BUDGET_ROUTE}${match.params.budgetId}${BUDGET_TRANSACTIONS_ROUTE}${meta.id}/view`,
+        search: location.search,
+      });
+    },
+    [history, match.params.budgetId, location.search],
+  );
+
+  const resultsFormatter = useMemo(() => getResultsFormatter(funds), [funds]);
+
+  return (
+    <Paneset>
+      <FiltersPane>
+        <SingleSearchForm
+          applySearch={applySearch}
+          changeSearch={changeSearch}
+          searchQuery={searchQuery}
+          isLoading={isLoadingTransactions}
+          areaLabelId="search.transactions"
+        />
+
+        <ResetButton
+          id="reset-transactions-filters"
+          reset={resetFilters}
+          disabled={!location.search}
+        />
+
+        <TransactionsFilters
+          activeFilters={filters}
+          applyFilters={applyFilters}
+        />
+      </FiltersPane>
+
+      <ResultsPane
+        title={resultsPaneTitle}
+        count={transactionsCount}
+      >
+        <MultiColumnList
+          id="transactions-list"
+          ariaLabel="Transactions"
```

Comment:
I'm not sure in this, maybe it should be `aria-label` and a translated string

Reply:
it shouldn't be `aria-label`, yep - a translated

Reply:
https://github.com/folio-org/stripes-components/blob/e904f1d1f36e18252274bc2d71b2a734c2056b57/lib/FilterPaneSearch/FilterPaneSearch.js#L36
https://github.com/folio-org/stripes-components/blob/e904f1d1f36e18252274bc2d71b2a734c2056b57/lib/FilterPaneSearch/FilterPaneSearch.js#L55
https://github.com/folio-org/stripes-components/blob/e904f1d1f36e18252274bc2d71b2a734c2056b57/lib/Datepicker/Calendar.js#L332
https://github.com/folio-org/ui-scan/blob/9fa075b4cf179c57a955feda2c62d5117072b4a6/CheckOut.js#L164
https://github.com/folio-org/ui-circulation/blob/c754ee7c1d2e54711a1a9b1771db73a089ff71b9/src/settings/components/CancelButton/CancelButton.js#L21
https://github.com/folio-org/ui-checkin/blob/0e04fbe43a92111615c16cc9fbe55317797f64d8/src/CheckIn.js#L342

Reply:
hm, I've seen this pros somewhere in MCL, I'll check

Reply:
it doesn't work here, no reason to pass